serverless/src/getAccessMethods/handler.js

```diff
@@ -79,9 +81,12 @@ const getAccessMethods = async (event, context) => {
     let onlineAccessFlag = false
     if (associatedGranules) {
       const { items: granuleItems } = associatedGranules
-      const [firstGranule] = granuleItems;
-
-      ({ onlineAccessFlag = false } = firstGranule)
+      onlineAccessFlag = getCollectionCapabilities({
```

Comment:
`getCollectionCapabilities` feels like overkill for this. We just need to find if any of the granuleItems have `online_access_flag = true`. I think this would work better
```
      const { items: granuleItems = [] } = associatedGranules
      const downloadable = granuleItems.some((granule) => {
        const { onlineAccessFlag = false } = granule
        return onlineAccessFlag
      })
```

Probably need to throw it into a sharedUtil because `actions/accessMethods.js` needs the same logic. On line 97 of that file it is still looking at tag data to determine if downloadable is true instead of looking at the granules.